tox.ini

```diff
@@ -39,7 +32,7 @@ deps =
         -rrequirements/requirements-testing.txt
 
 [testenv:dist]
-commands = ./runtests.py --no-pkgroot --staticfiles {posargs}
+commands = python -W error::DeprecationWarning -W error::PendingDeprecationWarning runtests.py --no-pkgroot --staticfiles {posargs}
```

Comment:
tox no longer allows `./runtests.py` to work, failing with:

```
dist: failed with ./runtests.py (resolves to ./runtests.py) is not allowed, use allowlist_externals to allow it
```

but we can use `python runtests.py` instead

I also added the warning flags as added to the main `[testenv]` in #7900